types/Config.js

```diff
@@ -42,6 +42,12 @@ class Config {
   /** @type {(interaction:Command, interaction: Discord.CommandInteraction, other: {setCoolDown(duration:number): void, [key:string|number]: any})=>boolean} */
   onInteraction = async () => { return true; };
 
+  /** @type {(eventName: "applicationCommandCreate" | "applicationCommandDelete" | "applicationCommandUpdate" | "channelCreate" | "channelDelete" | "channelPinsUpdate" | "channelUpdate" | "debug" | "emojiCreate" | "emojiDelete" | "emojiUpdate" | "error" | "guildBanAdd" | "guildBanRemove" | "guildCreate" | "guildDelete" | "guildIntegrationsUpdate" | "guildMemberAdd" | "guildMemberAvailable" | "guildMemberRemove" | "guildMembersChunk" | "guildMemberUpdate" | "guildUnavailable" | "guildUpdate" | "interaction" | "interactionCreate" | "invalidated" | "invalidRequestWarning" | "inviteCreate" | "inviteDelete" | "message" | "messageCreate" | "messageDelete" | "messageDeleteBulk" | "messageReactionAdd" | "messageReactionRemove" | "messageReactionRemoveAll" | "messageReactionRemoveEmoji" | "messageUpdate" | "presenceUpdate" | "rateLimit" | "ready" | "roleCreate" | "roleDelete" | "roleUpdate" | "shardDisconnect" | "shardError" | "shardReady" | "shardReconnecting" | "shardResume" | "stageInstanceCreate" | "stageInstanceDelete" | "stageInstanceUpdate" | "stickerCreate" | "stickerDelete" | "stickerUpdate" | "threadCreate" | "threadDelete" | "threadListSync" | "threadMembersUpdate" | "threadMemberUpdate" | "threadUpdate" | "typingStart" | "userUpdate" | "voiceStateUpdate" | "warn" | "webhookUpdate", args: [], other: {[key:string|number]: any})=>boolean} */
+  onEventBeforeChecks = async () => { return true; };
```

Comment:
Buna gerek yok zaten olan bir checkleme yok.

Reply:
var if(eventName == "") yaparken öneri veriyor

Reply:
Ondan bahsetmiyorum event olmadan önce checkleme birşey yok boşuna 2 tane koymaya gerek yok

Reply:
Diğeriyle aynı olsun diye böyle işler yapmana gerek yok yani

Reply:
heeeeeeeeeeeeeeeeeee anladım, aynısı interaction için de geçerli değil mi ki

Reply:
zaten peş peşe triggerlanıyor, beforecheck interaction la on interaction

Reply:
he pardon anladım, yetki zart zurt kontrolünden önce yapılıyor şimdi anladım tamam

Reply:
düzelttim